src/util/configInheritance.tsx

```diff
@@ -196,6 +197,15 @@ const getInstanceProfileProjectDefaults = (
     }
   }
 
+  if (configKey === "migration.stateful") {
```

Comment:
shall we leave a comment here to say this config can be set at the server, profile and instance level with that order of preference?